src/OmnichannelChatSDK.ts

```diff
@@ -81,6 +83,8 @@ class OmnichannelChatSDK {
     private ic3ClientLogger: IC3ClientLogger | null = null;
     private ocSdkLogger: OCSDKLogger | null = null;
     private isPersistentChat = false;
+    private isChatReconnect = false;
+    private isAuthenticated = false;
```

Comment:
We do not need to introduce this new attribute. We can determine if it's an auth'ed chat by checking if the value of `authenticatedUserToken` is not null